### PR TITLE
Added Notes to test case

### PR DESCRIPTION
**Pull request by zimarrio**

### CODE REVIEW

**Comment by Astro03** [comment]
Hi @zimarrio, THanks for the pull request. 

I'm working with TestRail 5.1.0.3413.  I did a "api/v2/get_case/661461" and do not see the "custom_testcase_notes" field in the json (did you mean "custom_notes".

The fields I saw when I did the above get command are as follows: "id","title","section_id","type_id","priority_id","milestone_id","refs","created_by","created_on","updated_by","updated_on","estimate","estimate_forecast","suite_id","custom_automated","custom_automated_thread_safe","custom_automated_name","custom_automated_extra_args","custom_preconds","custom_steps_separated","custom_steps","custom_expected","custom_notes"

Thanks,
ASTRO

File: TestRail/Types/Case.cs [hunk 1]
```diff
@@ -70,6 +74,7 @@ public static Case Parse(JObject json)
             c.Estimate = (string)json["estimate"];
             c.EstimateForecast = (string)json["estimate_forecast"];
             c.SuiteID = (ulong)json["suite_id"];
+            c.Notes = (string)json["custom_testcase_notes"];
```
Comment (Astro03) [inline TestRail/Types/Case.cs]: I do not see this field in the json when after a get_case query through the api

Comment (zimarrio) [inline TestRail/Types/Case.cs]: Hi @Astro03 , thanks for reviewing my pull request.
I have figured out what is happening.
The notes field is a custom field, that's why our names differ.
Do you think it's a good idea to have some kind of name value collection for the custom fields?

**Comment by Astro03** [comment]
sure, i'm okay with passing the json if that's what you need.. 

how do you populate the custom notes field?

**Comment by zimarrio** [comment]
The simplest way would be to have this property on Case.cs
public JObject JsonFromResponse { get; set; }
Then it would be accessed like this:
(string)testCase.JsonFromResponse["custom_testcase_notes"]
or we could create a method like this
testCase.GetCustomFieldValue<string>("custom_testcase_notes"]

File: TestRail/Types/Case.cs (hunk 1 above)
Comment (Astro03) [inline TestRail/Types/Case.cs]: were you going to make a pull request for this change? just curious.

Comment (Astro03) [inline TestRail/Types/Case.cs]: were you going to make a pull request for this change? just curious.

**Comment by Astro03** [comment]
completed with #18

**Comment by Astro03** [comment]
new version on NuGet is 1.0.0.10
https://www.nuget.org/packages/TestRail/

File: TestRail/Types/Case.cs (hunk 1 above)
Comment (zimarrio) [inline TestRail/Types/Case.cs]: I was waiting for your opinion about it. If you think it's worth it and what kind of implementation you think better suits this need.

Comment (Astro03) [inline TestRail/Types/Case.cs]: I've already checked it on and published I believe.   

I did the raw json from response. 

Ciao,
Astro
Sent from my phone

On Apr 4, 2016, at 03:34, zimarrio <notifications@github.com> wrote:

In TestRail/Types/Case.cs:

> @@ -70,6 +74,7 @@ public static Case Parse(JObject json)
>              c.Estimate = (string)json["estimate"];
>              c.EstimateForecast = (string)json["estimate_forecast"];
>              c.SuiteID = (ulong)json["suite_id"];
> +            c.Notes = (string)json["custom_testcase_notes"];
I was waiting for your opinion about it. If you think it's worth it and what kind of implementation you think better suits this need.

**Pull request closed without merging**